documentation/public/user-guide/07-semantic-analysis/02-binding-graph/index.md

```diff
@@ -0,0 +1,74 @@
+# 7.2. Binding Graph
+
+The binding graph is a graph structure that represents the relationships between identifiers across source files in a `CompilationUnit`.
```

Comment:
@ggiraldez I wonder if we should mention the `isIdentifier` API here in the examples?

Reply:
Oh, I forgot about that API. I'll add that.